README.md

```diff
@@ -7,7 +7,33 @@ An Android client for Open-Feeedback https://github.com/HugoGresse/open-feedback
 
 The SDK exposes a regular View and a Composable if you want to try [compose](https://developer.android.com/jetpack/compose). Please consult the [sample-app](sample-app/src/main/java/io/openfeedback/android/sample/MainActivity.kt) for usage.
```

Comment:
We might want to change that now 😄

Reply:
Oh yes. Maybe I can just remove it and add at the end of the Usage section, something like:
If you want to see an example, please check the sample app (+ link to the module)

Reply:
I as mainly mentioning the regular `View` thing that's not there anymore. Sample app can stay here or moved lower, no strong opinion there

Reply:
Done